pkg/testing/tools/git/git.go

```diff
@@ -26,17 +30,44 @@ type outputReader func(io.Reader) error
 // current repository ordered descending by creation date.
 // e.g. 8.13, 8.12, etc.
 func GetReleaseBranches(ctx context.Context) ([]string, error) {
-	c := exec.CommandContext(ctx, "git", "branch", "-r", "--list", "*/[0-9]*.*[0-9x]", "--sort=-creatordate")
+	c := exec.CommandContext(ctx, "git", "for-each-ref", "refs/heads/[0-9]*.*[0-9x]", "--format=%(refname:short)")
```

Comment:
```suggestion
	c := exec.CommandContext(ctx, "git", "for-each-ref", "refs/heads/[0-9]*.[0-9x]*", "--format=%(refname:short)")
```
just wondering if the second `*` be after the `[0-9x]`?!